src/parse.yy

```diff
@@ -1134,6 +1134,11 @@ specifier_qualifier_list
                       "function declarations.");
                 $$ = $2;
             }
+            else if ($1 == TYPEQUAL_VECTORCALL) {
+                Error(@1, "\"vectorcall\" qualifier is illegal outside of "
```

Comment:
I suggest adding test for that.

Reply:
The qualifier error mechanism doesn't work right now for any qualifiers. Will file a bug and add test sepaately.